examples/typescript/webpack.config.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+	module: {
+		rules: [
+			{
+				test: /\.tsx?$/,
+				loader: "ts-loader",
+				options: {
+					transpileOnly: true
+				}
+			}
+		]
+	},
+	resolve: {
+		extensions: [".ts"]
```

Comment:
```suggestion
		extensions: [".ts", ".js", ".json"]
```

Don't omit the default extensions otherwise stuff will break